File: prime.cpp
#include<iostream>
using namespace std;
int main(){

    int n;
    cout<<"Enter Your Number ";
    cin>>n;
    int i;
    for (i = 2; i < n; i++)
    {
        if (n%i==0)
        {
            cout<<"This is not a prime number."<<endl;
            break;
        }
        
    }
    if (i==n)
    {
        cout<<"This is a prime number."<<endl;
    }  

return 0;
}

File: Python Programs/AOT IT Workshop/prime.py
number = int(input("Enter the number to check: "))
number = int(number ** 0.5)
check = 1
for i in range(2,number):
    if number%i==0:
        print("The number is not prime")
        check=0
        break
if number==1:
    print("The number is not prime")
    check=0
if check:
    print("The number is prime")
